MaxMind.GeoIP2.UnitTests/DatabaseReaderTests.cs

```diff
@@ -77,12 +77,7 @@ public void InvalidCountryMethodForCityDatabase_ExceptionThrown()
             {
                 var exception = Record.Exception(() => reader.Country("10.10.10.10"));
                 Assert.NotNull(exception);
-                Assert.Contains(exception.Message,
-#if !NETCOREAPP1_1
-                    "A GeoIP2-City database cannot be opened with the Country method");
-#else
-                    "A GeoIP2-City database cannot be opened with the given method");
-#endif
+                Assert.Contains("A GeoIP2-City database cannot be opened with the", exception.Message);
```

Comment:
Simplified this assertion by removing the last varying part of the sentence as while netcoreapp2.0 has StackTrace (wasn't the case with STD 1.4 https://github.com/maxmind/GeoIP2-dotnet/blob/b2f4334980357c7269136688f8759002000e1220/MaxMind.GeoIP2/DatabaseReader.cs#L472-L481), JIT has improved the optimization and intermediate methods get effectively inlined and we instead get:

> A GeoIP2-City database cannot be opened with the Exception method